frontmacs-keys.el

```diff
@@ -17,6 +17,7 @@
 (require 'comment-dwim-2)
 (require 'browse-kill-ring)
 (require 'counsel)
+(require 'zoom-frm)
```

Comment:
How does `vendor` get added to the load-path? Wouldn't this need to be changed in `frontside.el` or `frontside-config.el`?

Reply:
That was going to be my next question. I'll update

Reply:
Does it work when you run init-test?